src/frontend/packages/core/src/core/entity-catalogue/entity-catalogue.service.ts

```diff
@@ -97,6 +97,12 @@ class EntityCatalogue {
       this.registerEntity(entity as StratosCatalogueEntity);
     }
   }
+
+  public clear() {
```

Comment:
I don't like the idea of having a clear method on the entity catalogue. Exposing this method publicly would open up the possibility of an extensions leaving the application in a broken state.

Is this just to avoid spamming the test output with the duplicate warning message?

Reply:
Correct, to avoid the duplicate warning. I'll take a look at other ways to do the same/similar